x-pack/plugins/enterprise_search/public/applications/shared/kibana/kibana_logic.test.ts

```diff
@@ -19,7 +19,7 @@ describe('KibanaLogic', () => {
 
   describe('mounts', () => {
     it('sets values from props', () => {
-      mountKibanaLogic(mockKibanaValues);
+      mountKibanaLogic(mockKibanaValues as any);
```

Comment:
suggestion: probably better to make mockKibanaValues match KibanaValues?

Reply:
The problem is `mountKibanaLogic` accepts `KibanaProps` which is slightly different from `KibanaValues`

The fields I added are on both so I'm not sure why this started erroring now. I spent 20 minutes trying to resolve it and then threw up my hands and add this `as any` since its a test file. I don't like this change either. I'll try to come back to this and see if I can resolve the type issue.